acrpull/deploy/templates/deployment.yaml

```diff
@@ -19,59 +19,48 @@ spec:
       securityContext:
         runAsNonRoot: true
       containers:
-        - command:
-            - /manager
-          args:
-            - "--health-probe-bind-address=:8081"
-            - "--metrics-bind-address=127.0.0.1:8080"
-            - "--leader-elect"
-          image: "{{ .Values.image }}"
-          name: acrpull-controller
-          ports:
-            - containerPort: 8080
-              protocol: TCP
-              name: metrics
-          securityContext:
-            runAsNonRoot: true
-            seccompProfile:
-              type: RuntimeDefault
-            allowPrivilegeEscalation: false
-            capabilities:
-              drop:
-                - "ALL"
-            readOnlyRootFilesystem: true
-            runAsUser: 1000
-            runAsGroup: 3000
-          livenessProbe:
-            httpGet:
-              path: /healthz
-              port: 8081
-            initialDelaySeconds: 15
-            periodSeconds: 20
-          readinessProbe:
-            httpGet:
-              path: /readyz
-              port: 8081
-            initialDelaySeconds: 5
-            periodSeconds: 10
-          resources:
-            limits:
-              cpu: 100m
-              memory: 100Mi
-            requests:
-              cpu: 100m
-              memory: 20Mi
+      - command:
+        - /manager
+        args:
+        - "--health-probe-bind-address=:8081"
+        - "--metrics-bind-address=127.0.0.1:8080"
+        - "--leader-elect"
+        - "--ttl-rotation-fraction={{ .Values.ttlRotationFraction }}"
+        image: "{{ .Values.image }}"
+        name: acrpull-controller
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+          name: metrics
+        securityContext:
+          runAsNonRoot: true
+          seccompProfile:
+            type: RuntimeDefault
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - "ALL"
+          readOnlyRootFilesystem: true
+          runAsUser: 1000
+          runAsGroup: 3000
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8081
+          initialDelaySeconds: 15
+          periodSeconds: 20
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: 8081
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        resources:
+          limits:
+            cpu: 100m
+            memory: 100Mi
+          requests:
+            cpu: 100m
+            memory: 20Mi
       serviceAccountName: acrpull
       terminationGracePeriodSeconds: 10
-      {{- with .Values.nodeSelector }}
```

Comment:
Deleting this was not strictly necessary, but having this (unused) functionality kept the file from being formatted by `ymlfmt`, so I figured we benefitted more from not including it.